src/test/java/org/kohsuke/github/GHContentIntegrationTest.java

```diff
@@ -260,6 +354,7 @@ public void testGetFileContentWithNonAsciiPath() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Why ignored?

Reply:
That's a mistake left over from when I was messing around with tests - sorry about that!